### PR TITLE
Update translations for 2.7 release

### LINKED ISSUE #4486: Prepare release 2.7.0
- [x] Resolve release blockers
  - [x] #4414
- [x] Re-generate VCR cassettes for API tests (see instructions on Taiga). You can find all test
      cases with `grep OFVCRMixin -r src`
  - [x] Accounts (`openforms.accounts.tests.test_oidc`)
  - Appoinments: Qmatic (`openforms.appointments.contrib.qmatic`) (no testenv available anymore)
  - Authentication plugins
    - [x] DigiD (Signicat): `openforms.authentication.contrib.digid.tests.test_signicat_integration`
    - [x] eHerkenning
          (Signicat):`openforms.authentication.contrib.eherkenning.tests.test_signicat_integration`
    - [x] DigiD/EH via OIDC: `openforms.authentication.contrib.digid_eherkenning_oidc`
    - [x] Org via OIDC: `openforms.authentication.contrib.org_oidc`
    - [x] `openforms.tests.test_registrator_prefill`
  - General purpose clients
    - ~~`openforms.contrib.brk`~~ test API is not operating as it should
    - [x] `openforms.contrib.haal_centraal.tests.test_integration`
    - [x] `openforms.contrib.kvk`
    - [x] `soap.tests.test_client`
    - `suwinet.tests.test_client` (testenv access has been retracted and won't be reinstated)
  - [x] Form imports: `openforms.forms.tests.test_import_export`
  - Payment plugins
    - [x] Ogone legacy: `openforms.payments.contrib.ogone.tests.test_client`
  - Registration plugins:
    - [x] Objects API: `openforms.registrations.contrib.objects_api`
    - [x] ZGW APIs: `openforms.registrations.contrib.zgw_apis`
- [x] Release new SDK version
- [x] Correct SDK version pinned in `.sdk-release`
- [x] Check translations
  - [x] SDK
  - [x] Backend
  - [x] Frontend
  - [x] Formio
- [x] Bump API version number
  - [x] Version bump
  - [x] Regenerate API spec
  - [x] Update READMEs with release dates + links
- [x] Bump version number (including package-lock.json)
- [x] Update changelog

### PR DESCRIPTION
**Pull request by sergei-maertens**
Partly closes #4486

**Changes**

* Updated backend translations
* Updated frontend translations

There will be some more changes in the formio-builder, but that's just re-compiling the translations and ship-it to be done :)

**Checklist**

Check off the items that are completed or not relevant.

- Impact on features

  - [x] Checked copying a form
  - [x] Checked import/export of a form
  - [x] Config checks in the configuration overview admin page
  - [x] Problem detection in the admin email digest is handled

- Release management

  - [x] I have labelled the PR as "needs-backport" accordingly

- I have updated the translations assets (you do NOT need to provide translations)

  - [x] Ran `./bin/makemessages_js.sh`
  - [x] Ran `./bin/compilemessages_js.sh`

- Commit hygiene

  - [x] Commit messages refer to the relevant Github issue
  - [x] Commit messages explain the "why" of change, not the how

### CODE REVIEW

File: src/openforms/conf/locale/nl/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -8860,250 +8873,284 @@ msgstr "Kan geen verbinding maken: {exception}"
...
-"2) type (the type of productaanvraag); 3) data (the submitted form data)"
+"UUID of the ProductAanvraag objecttype in the Objecttypes API. The "
+"objecttype should have the following three attributes: 1) submission_id; 2) "
+"type (the type of productaanvraag); 3) data (the submitted form data)"
 msgstr ""
-"URL naar het OBJECTTYPE voor de 'ProductAanvraag' in de Objecttypen API. Het "
-"objecttype moet de volgende attributen bevatten: 1) submission_id; 2) type "
+"UUID van het OBJECTTYPE voor de 'ProductAanvraag' in de Objecttypen API. Het"
+" objecttype moet de volgende attributen bevatten: 1) submission_id; 2) type "
 "(het type van de 'ProductAanvraag'); 3) data (ingezonden formulier gegevens)"
```
Comment (stevenbal) [inline src/openforms/conf/locale/nl/LC_MESSAGES/django.po:9061]:
```suggestion
"(het type van de 'ProductAanvraag'); 3) data (ingezonden formuliergegevens)"
```

**Review by stevenbal — CHANGES_REQUESTED** [review]

File: src/openforms/conf/locale/nl/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -1482,6 +1316,29 @@ msgstr "De volgende afspraakvelden dienen ingevuld te worden: {fields}"
 msgid "(updated)"
 msgstr "(bijgewerkt)"
 
+#: openforms/authentication/admin.py:56
+msgid "Means"
+msgstr "Middel"
+
+#: openforms/authentication/admin.py:60
+#| msgid "email subject"
```
[the remark this answers is not in the document]
Comment (sergei-maertens) [reply src/openforms/conf/locale/nl/LC_MESSAGES/django.po:1324]: ah this is automatically done with/from the fuzzy translations (where it took the message from)

File: src/openforms/conf/locale/nl/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -1773,140 +1675,273 @@ msgstr "Doorgaan als medewerker"
...
+#: openforms/authentication/models.py:200
+msgid ""
+"If a mandate is in play, then the mandate context must be provided. The "
+"details are tracked here, in line with the authentication context data JSON "
+"schema definition."
+msgstr ""
+"Als er sprake is van machtigen, dan moet een machtigingscontext opgegeven "
+"worden. De details van de machtiging worden hier opgeslagen in een formaat "
+"dat overeenkomt met het JSON Schema van het "
+"authenticatiecontextgegevensmodel."
```
[the remark this answers is not in the document]
Comment (sergei-maertens) [reply src/openforms/conf/locale/nl/LC_MESSAGES/django.po:1798]: for Germans, probably very natural :D

File: src/openforms/conf/locale/nl/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -3824,33 +3864,31 @@ msgstr "Bevestiging"
...
 
-#: openforms/emails/digest.py:247
+#: openforms/emails/digest.py:250
 msgid "BRK Client"
 msgstr "BRK client"
 
-#: openforms/emails/digest.py:254
-#, fuzzy
+#: openforms/emails/digest.py:257
 #| msgid "BRK Client"
```
[the remark this answers is not in the document]
Comment (sergei-maertens) [reply src/openforms/conf/locale/nl/LC_MESSAGES/django.po:3872]: not used, more like a hint :) I suppose these will be cleaned up if I run makemessages again, I'll give that a try

**Pull request merged**